README.md

```diff
@@ -3,21 +3,21 @@ Ethereum Network Package
 
 ![Run of the Ethereum Network Package](/run.gif)
 
-This is a Kurtosis Starlark Package that spins up an ethereum network.
+This is a Kurtosis Starlark Package that spins up an Ethereum network.
 
 ### Run 
 
-This assumes you have [Kurtosis CLI](https://docs.kurtosis.com/cli/) installed.
+This assumes you have the [Kurtosis CLI](https://docs.kurtosis.com/cli/) installed and the [Docker daemon](https://docs.kurtosis.com/install#i-install--start-docker) running on your local machine.
 
-Simply run 
+To get started, simply run:
 ```bash
 kurtosis run github.com/kurtosis-tech/eth-network-package
 ```
 
 ### Configuring the Network
 
-By default, this package spins up a single node with a [`geth`](https://github.com/kurtosis-tech/eth-network-package/blob/main/src/el/geth/geth_launcher.star) EL client and [`lighthouse`](https://github.com/kurtosis-tech/eth-network-package/blob/main/src/cl/lighthouse/lighthouse_launcher.star) CL client (more info on [node architecture](https://ethereum.org/en/developers/docs/nodes-and-clients/node-architecture/)) and comes with [five prefunded keys](https://github.com/kurtosis-tech/eth-network-package/blob/main/src/prelaunch_data_generator/genesis_constants/genesis_constants.star) for testing, but
-these and other parameters are configurable through a json file. 
+By default, this package spins up a single node with a [`geth`](https://github.com/kurtosis-tech/eth-network-package/blob/main/src/el/geth/geth_launcher.star) EL client and [`lighthouse`](https://github.com/kurtosis-tech/eth-network-package/blob/main/src/cl/lighthouse/lighthouse_launcher.star) CL client and comes with [five prefunded keys](https://github.com/kurtosis-tech/eth-network-package/blob/main/src/prelaunch_data_generator/genesis_constants/genesis_constants.star) for testing, but
+these and other parameters are configurable through a json file. Read more about the [node architecture here](https://ethereum.org/en/developers/docs/nodes-and-clients/node-architecture/)
```

Comment:
nit: could we add a period at end of sentence?

Reply:
Yes. Fixing now:

```suggestion
these and other parameters are configurable through a json file Read more about the [node architecture here](https://ethereum.org/en/developers/docs/nodes-and-clients/node-architecture/).
```